pages/1.13/security/oss/managing-users/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle:  Managing users
+title: Managing users
+menuWeight: 0
+excerpt: Managing users
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS can manage two types of users:
+
+* **External**: DC/OS never receives or stores the passwords of external users. Instead, it delegates the verification of the user’s credentials to the Auth0 identity provider instance managed by Mesosphere. Only the user ID of one of the three available login providers Google, GitHub or Microsoft is stored inside DC/OS.
+* **Local**: Local user accounts exist inside the DC/OS cluster.
+
+Additional users can be added create via the web interface, [DC/OS CLI](/1.13/cli) or via the [IAM API](/1.13/security/oss/iam-api/) depending on the user type.
```

Comment:
"added create"

Reply:
When you start the sentence with "Additional users" then the reader wonders "additional to what"?

Maybe just "Users can be added ..."?